From e483571e5d9658381e8342b2c89d3489c5815d0f Mon Sep 17 00:00:00 2001
From: Vahempio <vahempio@gmail.com>
Date: Fri, 29 Jun 2018 19:17:40 +0200
Subject: [PATCH 24/35] MAC: add new Murata range

New herolte OUI observed: 1c:70:22
Checked on hwaddress.com

Change-Id: I4a2c15e315b46af81f815114cf85f0d1cced87f3
(cherry picked from commit ba6c738c2f39fc1011ae5f61067f84b1bf430111)
---
 macloader/macaddr_mappings.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/macloader/macaddr_mappings.h b/macloader/macaddr_mappings.h
index 96786e7..b5c60a6 100644
--- a/macloader/macaddr_mappings.h
+++ b/macloader/macaddr_mappings.h
@@ -54,6 +54,7 @@ static const struct company_range murata_ranges = {
         "10:a5:d0",
         "10:d5:42",
         "14:7d:c5",
+        "1c:70:22",
         "1c:99:4c",
         "20:02:af",
         "40:f3:08",
-- 
2.17.1

